lib/inflastructure/mock/mock_counter_repository.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:count_habits/domain/counter/counter_repository.dart';
+import 'package:count_habits/domain/counter/entity/counter.dart';
+
+class MockCounterRepository implements CounterRepository {
+  @override
+  Future<void> delete(int id) {
+    // TODO: implement delete
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<List<Counter>> fetchAll() {
+    // TODO: implement fetchAll
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<Counter> update({required int id, required String name}) {
+    // TODO: implement update
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kolbe-ryo/count_habits/blob/1869c56190813e5f789c063df231c4d4c582c775/lib/inflastructure/mock/mock_counter_repository.dart#L19"></span>To-do comment doesn't follow the Flutter style</td>
    </tr>
  </tbody>
</table>